src/permissions/PermissionController.ts

```diff
@@ -220,6 +220,14 @@ export type RequestPermissions = {
   handler: GenericPermissionController['requestPermissions'];
 };
 
+/**
+ * Removes for each origin removes given permissions
```

Comment:
Perhaps reword to be a bit more clear?
"Revokes specified permissions from specified origin" or something like that?